exercises/practice/grains/.docs/instructions.append.md

```diff
@@ -0,0 +1,3 @@
+#### Independent of other languages
```

Comment:
```suggestion
# Independent of other languages
```

My understanding is that Exercism ignores this header when appending this document's contents to the instructions. So the student won't see the header text. So if you want the header text to be visible, I think you need to add another header underneath the first one.